canonical-web-and-design/microk8s.io
# Revamp install journey

## Done

Updated the install journey

## QA

- Check out this branch
- `./run` the project
- Visit http://0.0.0.0:8027/
- See that the page matches the [design](https://github.com/canonical-web-and-design/web-squad/issues/1828)
- See that the OS selector works, and the installation instructions make sense

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1828
